extensions/amp-story-360/0.1/amp-story-360.js

```diff
@@ -138,7 +144,7 @@ class CameraOrientation {
   static fromDegrees(heading, pitch, zoom) {
     return new CameraOrientation(
       deg2rad(-pitch - 90),
```

Comment:
you can probably use it here too :)

Reply:
Good call. Since it's `CENTER_OFFSET` I agree it makes sense for pitch and heading.